fenix/app/src/main/res/values/strings.xml

```diff
@@ -478,11 +478,15 @@
     <!-- Preference for accessibility -->
     <string name="preferences_accessibility">Accessibility</string>
     <!-- Preference to override the Firefox Account server -->
-    <string name="preferences_override_fxa_server">Custom Firefox Account server</string>
+    <string name="preferences_override_fxa_server" moz:RemovedIn="120" tools:ignore="UnusedResources">Custom Firefox Account server</string>
+    <!-- Preference to override the Mozilla Account server -->
+    <string name="preferences_override_account_server">Custom Mozilla Account server</string>
```

Comment:
"account" should almost always be lowercase, except in "“Mozilla Account Privacy Policy” and “Mozilla Account Terms of Service” (anytime other navigation items are using headline casing, for consistency)
Content guidelines: https://www.figma.com/file/7nsOBSt3VTeCimDqavQT9J/Mozilla-Accounts-Content-Guidelines?type=design&node-id=1903-403&mode=design&t=gtC3OzSNcm5JK5b5-0